From sd@sedf.de  Fri Nov  6 14:58:11 2009
From: Sebastian Dalfuﬂ <sd@sedf.de>
Date: Fri, 6 Nov 2009 14:49:28 +0100
Subject: Staging: rt2860: remove superfluous newlines
To: greg@kroah.com
Message-ID: <20091106134928.GC27210@sedf.de>

From: Sebastian Dalfuﬂ <sd@sedf.de>

This patch is based on next-20091106.
This tiny patch removes a few quite unnecessary extra newlines from
DBGPRINT() and printk() strings.

Signed-off-by: Sebastian Dalfuﬂ <sd@sedf.de>
Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/rt2860/common/rtmp_init.c |    2 +-
 drivers/staging/rt2860/common/rtmp_mcu.c  |    2 +-
 drivers/staging/rt2860/rt_linux.c         |    2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

--- a/drivers/staging/rt2860/common/rtmp_init.c
+++ b/drivers/staging/rt2860/common/rtmp_init.c
@@ -181,7 +181,7 @@ int RTMPAllocAdapterBlock(void *handle,
 		}
 		pAd->BeaconBuf = pBeaconBuf;
 		DBGPRINT(RT_DEBUG_OFF,
-			 ("\n\n=== pAd = %p, size = %d ===\n\n", pAd,
+			 ("=== pAd = %p, size = %d ===\n", pAd,
 			  (u32)sizeof(struct rt_rtmp_adapter)));
 
 		/* Init spin locks */
--- a/drivers/staging/rt2860/common/rtmp_mcu.c
+++ b/drivers/staging/rt2860/common/rtmp_mcu.c
@@ -163,7 +163,7 @@ int RtmpAsicLoadFirmware(struct rt_rtmp_
 
 	if (Index > 1000) {
 		DBGPRINT(RT_DEBUG_ERROR,
-			 ("NICLoadFirmware: MCU is not ready\n\n\n"));
+			 ("NICLoadFirmware: MCU is not ready\n"));
 		Status = NDIS_STATUS_FAILURE;
 	}
 
--- a/drivers/staging/rt2860/rt_linux.c
+++ b/drivers/staging/rt2860/rt_linux.c
@@ -340,7 +340,7 @@ int RTMPAllocateNdisPacket(struct rt_rtm
 	if (pPacket == NULL) {
 		*ppPacket = NULL;
 #ifdef DEBUG
-		printk("RTMPAllocateNdisPacket Fail\n\n");
+		printk("RTMPAllocateNdisPacket Fail\n");
 #endif
 		return NDIS_STATUS_FAILURE;
 	}
